src/components/ProgressIndicator/_progress-indicator.scss

```
@@ -1,5 +1,36 @@
 @import '~carbon-components/scss/components/progress-indicator/progress-indicator';
+@import '../../globals/vars';
 
-.bx--skeleton .bx--progress-step {
-  flex: unset;
+.#{$iot-prefix}--progress-indicator {
+  .bx--skeleton .bx--progress-step {
+    flex: unset;
+  }
+}
+
+html[dir='rtl'] {
+  .#{$iot-prefix}--progress-indicator {
+    .bx--progress-step svg {
+      margin-right: 0;
+      margin-left: 0.5rem;
```

Where possible, these can be modified to use the [carbon spacing scale variables](https://www.carbondesignsystem.com/guidelines/spacing#spacing-scale)

```suggestion
      margin-left: $spacing-03;
```